packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appatidp.md

```diff
@@ -3,23 +3,19 @@ When you're configuring federation between two Okta orgs, use OpenID Connect as
 1. In the Admin Console for the Okta org that represents the Identity Provider, go to **Applications** > **Applications**.
 1. Click **Create App Integration**.
 1. On the Create a new app integration page, select **OIDC - OpenID Connect** as the **Sign-in method**. You need a trusted client, so select **Web Application** as the **Application type**.
-1. Click **Next**.
-1. Enter a name for your application.
-1. In the **Sign-in redirect URIs** box, enter the redirect URI. The redirect URI:
-    * Is constructed using your Okta org subdomain and the callback endpoint
-    For example, if your Okta subdomain is called `company`, then the URI would be: `https://company.okta.com/oauth2/v1/authorize/callback`. If you have configured a custom domain in your Okta Org, use that value to construct your redirect URI, such as `https://login.company.com/oauth2/v1/authorize/callback`.
-    * Is where the IdP returns the authentication response (the access token and the ID token)
-    * Needs to be a secure domain that you own
-    * Should match the redirect URI sent in the authorize request from the client
+1. Click **Next** and enter a name for your application.
+1. In the **Sign-in redirect URIs** box, enter the redirect URI taking into consideration the following:
+    * Constructed using your Okta org subdomain and the callback endpoint.
+    For example, if your Okta subdomain is called `company`, then the URI would be: `https://company.okta.com/oauth2/v1/authorize/callback`. If you’ve configured a custom domain in your Okta Org, use that value to construct your redirect URI, such as `https://login.company.com/oauth2/v1/authorize/callback`.
+    * Defined as where the IdP returns the authentication response (the access token and the ID token).
+    * Needs to be a secure domain that you own.
+    * Matches the redirect URI sent in the authorize request from the client.
 
-    Include all base domains (Okta domain and custom domain) that your users will interact with in the allowed redirect URI list.
+    Include all base domains (Okta domain and custom domain) that your users interact with in the allowed redirect URI list.
```

Comment:
```suggestion
    It include all base domains (Okta domain and custom domain) that your users interact with in the allowed redirect URI list.
```

Reply:
Suggesting these additions to make the list of items complete sentence, so that you can take them into consideration (from the stem)